src/App.jsx

```diff
@@ -3,26 +3,57 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    this.loadData();
+  }
+
+  loadData = async() => {
+    // eslint-disable-next-line
+    const todos = await getTodos(this.props.userId);
```

Comment:
why do you pass userId to getTodos? you need all todos
also App doesn't have props